pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pydantic-avro"
-version = "0.3.1"
+version = "0.3.2"
```

Comment:
Please revert this, I should do a new release for this

Reply:
Fixed.